cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -43,13 +48,16 @@ def config_manager(config_manager_obj):
 
 
 @pytest.fixture(scope="function")
-def catalog_item(appliance, request, config_manager, dialog, catalog):
+def catalog_item(appliance, request, config_manager, dialog, catalog, job_type):
     config_manager_obj = config_manager
     provider_name = config_manager_obj.yaml_data.get('name')
     provisioning_data = config_manager_obj.yaml_data['provisioning_data']
-    item_type, provider_type, template = map(provisioning_data.get,
+    if job_type == 'template':
+        item_type, provider_type, template = map(provisioning_data.get,
                                             ('item_type', 'provider_type', 'template'))
-
+    else:
+        item_type, provider_type, template = map(provisioning_data.get,
+                                            ('item_type', 'provider_type', 'workflow'))
```

Comment:
Why not just pass `job_type` here? This will reduce LOC.

Reply:
^ **Optional**

Reply:
I'm actually going to call this **Required** - this is unnecessary logic and will just further complicate when we get a 3rd parameter.  Just pass `job_type` to the map call directly.

Reply:
Done